src/parsing/Parse_target.ml

```diff
@@ -97,6 +97,14 @@ let just_parse_with_lang lang file =
   | _else_ -> !just_parse_with_lang_ref lang file
   [@@profiling]
 
+let lang_supports_implicit_return (lang : Lang.t) =
```

Comment:
This should probably be put in the `Implicit_return` module.

Reply:
Done.